app/src/tests/mocks/mockResponses.ts

```diff
@@ -1,30 +1,23 @@
-export const processFilesMockResponseNonConflict = {
-  result: "file"
-};
-
-export const processFilesMockResponseConflict = {
+export const mergeFilesMockResponse = {
```

Comment:
explicitly declare this as SpecMathMergeResponse? (as you do with the mock request)

Reply:
I missed this one, it should be mergeSpecsMockResponse as it is with the request